File: packages/insomnia-httpsnippet/test/fixtures/output/csharp/restsharp/full.cs
var client = new RestClient("http://mockbin.com/har?foo=bar&foo=baz&baz=abc&key=value");
var request = new RestRequest(Method.POST);
request.AddHeader("content-type", "application/x-www-form-urlencoded");
request.AddHeader("accept", "application/json");
request.AddCookie("foo", "bar");
request.AddCookie("bar", "baz");
request.AddParameter("application/x-www-form-urlencoded", "foo=bar", ParameterType.RequestBody);
IRestResponse response = client.Execute(request);

File: packages/insomnia-httpsnippet/test/fixtures/output/javascript/jquery/full.js
var settings = {
  async: true,
  crossDomain: true,
  url: 'http://mockbin.com/har?foo=bar&foo=baz&baz=abc&key=value',
  method: 'POST',
  headers: {
    cookie: 'foo=bar; bar=baz',
    accept: 'application/json',
    'content-type': 'application/x-www-form-urlencoded'
  },
  data: {
    foo: 'bar'
  }
};

$.ajax(settings).done(function(response) {
  console.log(response);
});
